### ISSUE TITLE
Feature: Command line launch option that passes to command palette

### ISSUE THREAD

**Issue by sgoley** [microsoft/vscode#101933]
TLDR: Command line launch option that passes to command palette
`vscode://c:/myProject/ --palette '> Python: Create Terminal' `

References:

Full Docs on Command Line capabilities: https://code.visualstudio.com/docs/editor/command-line
Specifics around opening with URLs: https://code.visualstudio.com/docs/editor/command-line#_opening-vs-code-with-urls

Description:
It would be very useful as above to pass command palette commands to the vs code path variable.  

Examples:
`vscode://c:/myProject/ --palette '> Python: Create Terminal' `
`vscode://test_file/c:/myProject/ --palette '> Python: Run Current Test File' `
`vscode://c:/LaTeX_Project/ --palette '> LaTeX Workshop: Build LaTeX Project' `
and many many more.
